api/krusty/kustomizer.go

```diff
@@ -112,10 +112,16 @@ func (b *Kustomizer) Run(
 	}
 	m.RemoveBuildAnnotations()
 	if !utils.StringSliceContains(kt.Kustomization().BuildMetadata, types.OriginAnnotations) {
-		m.RemoveOriginAnnotations()
+		err = m.RemoveOriginAnnotations()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
@natasha41575 can you confirm you weren't ignoring these errors on purpose? It would be very unusual and likely beyond the user's control for this to fail, but I'd be inclined to think we should handle them regardless.

Reply:
Ah no, I wasn't ignoring them on purpose and I'm surprised the linter didn't complain about this. I'm in support of handling the errors.